examples/test_bert_ptq_cpu.py

```diff
@@ -20,6 +20,7 @@ def setup():
 
 
 def check_output(footprint):
+    footprint = footprint[0]
```

Comment:
Could you help explain how the accelerators limit the checking only on first footprint node?

Reply:
will change to loop